app/processing/statement/get-statement.js

```diff
@@ -0,0 +1,26 @@
+const { getDetails, getAddress, getDetailedFunding, getScheme, getDetailedPayments } = require('./components')
+const getCalculation = require('../calculation')
+const getPaymentRequest = require('../payment-request')
+const getSettlement = require('../settlement')
+
+const getStatement = async (settlementId) => {
+  const settlement = await getSettlement(settlementId)
+  const paymentRequestId = settlement.paymentRequestId
+  const paymentRequest = await getPaymentRequest(paymentRequestId)
+  const calculation = await getCalculation(paymentRequestId)
+  const details = await getDetails(calculation.sbi)
+  const address = await getAddress(calculation.sbi)
+  const detailedFunding = await getDetailedFunding(calculation.calculationId)
+  const scheme = await getScheme(paymentRequest)
+  const payments = await getDetailedPayments(calculation, paymentRequest, settlement)

```

Comment:
Same comment here

Reply:
same as  above:
https://github.com/DEFRA/ffc-pay-statement-constructor/blob/PAY-2318-aggregate-map-and-build-statement-constructor-object/app/processing/statement/components/get-details.js